apps/frontend/src/pages/project/project.tsx

```diff
@@ -59,6 +58,11 @@ const Project = (): JSX.Element => {
 		projectStatus,
 	} = useAppSelector(({ projects }) => projects);
 
+	const { authenticatedUser, projectUserPermissions, userPermissions } =
+		useAppSelector(({ auth }) => auth);
+
+	const allPermissions = [...userPermissions, ...projectUserPermissions];
```

Comment:
we don't need to combine permissions here, we need to check root permissions, and then check project permissions. If one of the check is true, then display needed functionality
```suggestion
```